procmon/main.cpp

```diff
@@ -1131,9 +1177,6 @@ static void messageThread(MonitorConfig config)
 
     string cmd = "fuser -k " + port + "/tcp >/dev/null 2>&1";
```

Comment:
This will work as not-root as long as the target processes are owned by the current user.